network-api/networkapi/donate/pagemodels/base_page.py

```diff
@@ -1,8 +1,8 @@
 from wagtail.models import Page
 
 from networkapi.wagtailpages.models import FoundationMetadataPageMixin
+from networkapi.wagtailpages.pagemodels.mixin.foundation_navigation import FoundationNavigationPageMixin
 
-
-class BaseDonationPage(FoundationMetadataPageMixin, Page):
+class BaseDonationPage(FoundationMetadataPageMixin,FoundationNavigationPageMixin, Page):
```

Comment:
Needed to add this mixin so the navigation bar will accurately show the links at the top of the page. EG: “[Who we are](https://foundation.mozilla.org/en/who-we-are/), [What we do](https://foundation.mozilla.org/en/what-we-do/), [What we fund](https://foundation.mozilla.org/en/what-we-fund/), [What you can do](https://foundation.mozilla.org/en/what-you-can-do/), [Blog](https://foundation.mozilla.org/en/blog/)”?